packages/ui/components/ui/select-account.tsx

```diff
@@ -38,17 +39,21 @@ export const SelectAccount = ({ getAddrByIndex }: SelectAccountProps) => {
         <div className='flex w-full flex-col'>
           <AccountSwitcher account={index} onChange={setIndex} />
 
-          <div className='mt-4 flex items-center justify-between gap-1 break-all rounded-lg border bg-background px-3 py-4'>
-            <div className='flex items-center gap-[6px] overflow-hidden'>
-              <div className='shrink-0'>
-                <AddressIcon address={address} size={24} />
-              </div>
+          <div className='mt-4'>
+            <Box>
+              <div className='flex items-center justify-between gap-1 break-all'>
```

Comment:
Review this with whitespace hidden. I added an `mt-4` wrapper above the box, then removed the `<Box />`-specific classes from this div.